docs/visualisation/alchemlyb.visualisation.plot_dF_state.rst

```diff
@@ -0,0 +1,33 @@
+.. _visualisation_plot_dF_state:
+
+Plot dF states from multiple estimators
+=======================================
+
+The function :func:`~alchemlyb.visualisation.plot_dF_state` allows the user to
+plot and compare the dF from various kinds of :class:`~alchemlyb.estimators`.
+
+To compare the dF states of a single alchemical transformation among various
+:class:`~alchemlyb.estimators`, the user can pass a list of `dhdl_data`. (e.g.
```

Comment:
@orbeckst Thank you for the review. I was hoping to show a list with reference to different estimators instead of a line of code.
``estimators` = [:class:`~alchemlyb.estimators.TI`, :class:`~alchemlyb.estimators.BAR`, :class:`~alchemlyb.estimators.MBAR`]`
I would imagine that doing this would not allow the user to click TI to show the page of alchemlyb.estimators.TI